crash_similarity.py

```diff
@@ -70,6 +70,18 @@ def get_stack_trace_from_crashid(crash_id):
 def get_stack_traces_for_signature(fnames, signature):
     traces = set()
 
+    #query stack traces online
+    url = 'https://crash-stats.mozilla.com/api/SuperSearch'
+    params = {
+        'signature': '=' + signature,
+        '_columns':  'proto_signature'
+    }
```

Comment:
Instead of using `_columns`, which will also return duplicate results, we could use `_facets`: https://crash-stats.mozilla.com/documentation/supersearch/api/#param-_facets.
The default number of facets returned is 50, we can increase it using the `_facets_size` parameter: https://crash-stats.mozilla.com/documentation/supersearch/api/#param-_facets_size.